src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,55 @@
+@import './variables';
+
+@mixin product-title-font {
+  font-family: $card-ff;
+  color: $text-color;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
```

Comment:
Consider using a more descriptive mixin name that reflects the specific usage or thematic grouping of the styles. For example, if this mixin is used for titles within a product card, `product-title-font` might be too generic.